lib/ansible/modules/system/mount.py

```diff
@@ -236,6 +244,8 @@ def set_mount(module, args):
         changed = True
 
     if changed and not module.check_mode:
+        if module.params['backup']:
+            module.backup_local(args['fstab'])
```

Comment:
I'd move this to the `write_fstab` function so it does not need to be repeated.

Reply:
Done.